generate.js

```diff
@@ -47,8 +47,15 @@ const template = `
     <div class="orgs">
       {{#orgs}}
         <div class="org">
-          <h3><a href="https://codein.withgoogle.com/organizations/{{slug}}">{{name}}</a></h3>
-          <p>Task Completed: {{completed_task_instance_count}}</p>
+          <h3>
+            <a href="https://codein.withgoogle.com/organizations/{{slug}}">{{name}}</a>
+            <p>Task Completed: {{completed_task_instance_count}}</p>
+            {{#github}}
+            <a href="https://github.com/{{github}}">
+              <img src="https://image.flaticon.com/icons/png/512/25/25231.png" height="14" />
```

Comment:
Is this icon "safe" to use without putting a credit?

Reply:
That's a good point. IANAL but I'd assume so because it's GitHub's logo.

Reply:
I should be able to use the official GitHub icon, right?

https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png